test/src/api/responder04Test.php

```diff
@@ -263,11 +265,11 @@ public function test_interface_game_00061() {
 
         $this->verify_api_submitTurn(
             array(),
-            'responder003 chose to perform a Skill attack using [p(V=9)!:3] against [(C):6S]; responder003 must turn down fire dice to complete this attack. ',
+            'responder003 chose to perform a Skill attack using [p(V=9)!:2] against [(C):6S]; responder003 must turn down fire dice to complete this attack. ',
             $retval, array(array(0, 1), array(1, 1)),
             $gameId, 1, 'Skill', 0, 1, '', array());
```

Comment:
So this looks good; you just want to verify that the attack is rejected rather than accepted.  Try using `verify_api_submitTurn_failure()`, for which there's other examples in the tests you can copy from.

Reply:
And then afer this, you add another `verify_api_submitTurn()` call where the attacking dice are `array(0, 0)` and `array(0, 1)`, and the defending die is `array(1, 0)`.

And at that point you should be able to submit a successful fire turndown.  Like i said before, i can't think of how to put the lack of internal error in response to an invalid fire turndown, in this test, but hopefully we can validate it some other way?